setup.cfg

```diff
@@ -1,63 +1,6 @@
-[metadata]
-name = qcodes
-maintainer = QCoDeS Core Developers
-maintainer_email = qcodes-support@microsoft.com
-description = Python-based data acquisition framework developed by the Copenhagen / Delft / Sydney / Microsoft quantum computing consortium
-long_description = file: README.rst
-long_description_content_type = text/x-rst
-url = https://github.com/QCoDeS/Qcodes
-classifiers =
-    Development Status :: 3 - Alpha
-    Intended Audience :: Science/Research
-    License :: OSI Approved :: MIT License
-    Programming Language :: Python :: 3 :: Only
-    Programming Language :: Python :: 3.7
-    Programming Language :: Python :: 3.8
-    Programming Language :: Python :: 3.9
-    Programming Language :: Python :: 3.10
-    Topic :: Scientific/Engineering
-license = MIT
-project_urls =
-    Documentation = https://qcodes.github.io/Qcodes/
-    Source = https://github.com/qcodes/qcodes
-    Tracker = https://github.com/QCoDeS/Qcodes/issues
-    Changelog = https://qcodes.github.io/Qcodes/changes/index.html
-
 [options]
```

Comment:
do you have a feeling on when the options section will be added to pyproject.toml?

Reply:
They are there but the syntax is not officially supported yet. No eta as far as I know